src/frontends/pytorch/src/op/list_construct.cpp

```diff
@@ -24,7 +26,8 @@ OutputVector translate_list_construct(NodeContext& context) {
         if (c_node->get_shape().size() == 0) {
             c_node = std::make_shared<v0::Constant>(c_node->get_element_type(), Shape{1}, c_node->get_data_ptr());
         }
-        consts.push_back(c_node);
+        auto unsqueezed_c_node = context.mark_node(std::make_shared<v0::Unsqueeze>(c_node, const_0));
```

Comment:
Why is this needed? The code on lines 2-28 ensures all constants are at least 1D, and if you do `Unsqueeze`, you will make them 2D, which is incorrect in case when `prim::ListConstruct` is used for scalars.

Reply:
I added this `Unsqueeze` operator, because after that there is a `Concat` operator. Without this `Unsqueeze` for inputs like [1,2],[3,4] we will obtain [1,2,3,4] which doesn't look like a list of tensors. I think that result [[1,2],[3,4]] (which is obtained by adding this `Unsqueeze`) would be much more appropriate. On the other hand, I didn't notice that there is a special case for scalars and I agree with your comment. Maybe it would be enough to put `Unsqueeze` after the `else` statement?